logistics_planning_invoicing/wizards/logistics_schedule_account_move_wizard.py

```diff
@@ -0,0 +1,28 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import fields, models, _
+
+
+class LogisticsScheduleAccountMove(models.TransientModel):
+    _name = 'logistics.schedule.account.move.wizard'
+
+    logistics_schedule_ids = fields.Many2many('logistics.schedule')
+    company_id = fields.Many2one('res.company', related='logistics_schedule_ids.company_id')
+    carrier_id = fields.Many2one('res.partner')
+    journal_id = fields.Many2one('account.journal', related='company_id.logistics_schedule_default_journal_id', readonly=False)
```

Comment:
This is wrong, this field should be independent from default one, only filled by default. With this code, if you change it, you're actually changing the default journal permanently, and such action should be only made from settings.